tests/acceptance/DatasetView.feature

```diff
@@ -143,3 +143,29 @@ Feature: a user visit the dataset page
     When I am on "/dataset/100935"
     Then I should see "Github links"
     And I should see "https://github.com/cihga39871/Atria"
+
+  @ok @issue-1056
```

Comment:
In a few months time, I will have forgotten what issue 1056 was about. I think a better tag to use would be something like `@cite-dataset-button`